gym_minigrid/envs/negated_goals.py

```
@@ -60,11 +66,28 @@ def get_vocabulary(self):
                     self.vocabulary.add(word)
         return self.vocabulary
 
+    def set_train(self):
+        self.mode = "TRAIN"
+
+    def set_eval(self):
+        self.mode = "EVAL"
```

Since we'll have more kinds of splits in the future, I would combine these functions to a `set_mode(self, mode):` function. This function should also raise an error if the mode isn't a valid mode type (easier to debug if the error is thrown while setting the mode rather than when computing indices).

---

Done.